src/ripple/app/misc/ValidatorList.h

```diff
@@ -433,13 +445,16 @@ ValidatorList::onConsensusStart (
         }
     }
 
-    if (minimumQuorum_ && (seenValidators.empty() ||
-            rankedKeys.size() < quorum))
+    if (specifiedQuorum_)
```

Comment:
This changes how the `--quorum` command line argument works.
Previously (including pre-dynamic UNL), this specified quorum would only be used if rippled determined that its safe quorum was unattainable.
Specifying `--quorum` is useful when bootstrapping a new set of validators or when running a validator that you know others are trusting but isn't on a published list.
I think we should keep this check as:
```
if (specifiedQuorum_ && seenValidators.size() < quorum)
```